tests/foreman/ui/test_settings.py

```diff
@@ -499,8 +499,9 @@ def test_negative_update_hostname_with_empty_fact(session, setting_update):
     new_hostname = ""
     property_name = setting_update.name
     with session:
-        response = session.settings.update(property_name, new_hostname)
-        assert response is not None, "Value can't be blank"
+        with pytest.raises(AssertionError) as context:
+            session.settings.update(property_name, new_hostname)
+            assert is_valid_error_message(str(context.value))
```

Comment:
this is the only usage in the code of `is_valid_error_message` that I have found, so if not explained or not in future plans, please consider comparing it directly here

And right now the method compares error value with one of the three error values, it doesn't seem as a good concept, to find whether an error is one of such, IMO error should be compared directly

Reply:
three error values make more sense if each parametrized value provides a different error :)

Reply:
@tstrych `is_valid_error_message` and `valid_error_messages` only used once so I removed these and compared the failures in test itself.